test/tests_for_integration/api_to_db_test.py

```diff
@@ -448,8 +457,8 @@ def _get_condor_sub_for_rj_param_set(
             "+KB_PARENT_JOB_ID": f'"{parent_job_id}"',
             "+KB_MODULE_NAME": '"mod"',
             "+KB_FUNCTION_NAME": '"meth"',
-            "+KB_APP_ID": f'"{_APP}"',
-            "+KB_APP_MODULE_NAME": '"mod"',
+            "+KB_APP_ID": f'"{app_id}"' if app_id else "",
```

Comment:
I wonder if an empty string without quotes works in the else condition. I think the simplest way to test it out is to submit the job and see what happens to it.

Reply:
If including quotes is the normal way of doing things, I'd rather just update the code.